cogboard-webapp/src/components/widgets/types/TextWidget.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import Typography from '@material-ui/core/Typography';
+import {Requireable as string} from "prop-types";
+
+const TextWidget = props => {
+  const { text, textSize } = props;
+  return (
+      <Typography variant={textSize}>
+        <pre style={{ fontFamily: 'inherit' }}>

```

Comment:
is `pre` required?

Reply:
same as above, will be changed soon by @goeson00